query_execution/QueryContext.cpp

```diff
@@ -65,6 +68,13 @@ QueryContext::QueryContext(const serialization::QueryContext &proto,
                                                         storage_manager));
   }
 
+  for (int i = 0; i < proto.bloom_filters_size(); ++i) {
+    const serialization::BloomFilter &bloom_filter_proto = proto.bloom_filters(i);
+    bloom_filters_.emplace_back(new BloomFilter(bloom_filter_proto.bloom_filter_seed(),
```

Comment:
Could we have a `BloomFilter` constructor that takes its proto as the single argument? Thus, this line could be simplified to

`bloom_filters_.emplace_back(new BloomFilter(bloom_filter_proto));`